File: Developer_code/03-angular-springboot-ecommerce/Ecommerce backend/Ecommerce/src/main/java/com/luv2code/springboot/Ecommerce/dto/Purchase.java
package com.luv2code.springboot.Ecommerce.dto;

import com.luv2code.springboot.Ecommerce.entity.Address;
import com.luv2code.springboot.Ecommerce.entity.Customer;
import com.luv2code.springboot.Ecommerce.entity.Order;
import com.luv2code.springboot.Ecommerce.entity.OrderItem;
import jakarta.transaction.UserTransaction;
import lombok.Data;

import java.util.Set;

@Data
public class Purchase {

    private Order order;
    private Customer customer;
    private Address shippingAddress;
    private Address billingAddress;
    private Set<OrderItem> orderItem;
}

File: Developer_code/03-angular-springboot-ecommerce/Angular Ecommerce/angular-ecommerce/src/app/commons/purchase.ts
import { Address } from "./address";
import { Customer } from "./customer";
import { Order } from "./order";
import { OrderItem } from "./order-item";

export class Purchase {

    order !:Order;
    orderItem !:OrderItem[];
    customer !:Customer
    shippingAddress !:Address;
    billingAddress !:Address;
}
